### PR TITLE
feat: JS performance docs

### PR DESCRIPTION
**Pull request by HazAT**
This moved part of the JS specific performance documentation into the JavaScript section.

To make what's written in the `Filter out unwanted Spans` section 100% true, we need to release this PR first.
https://github.com/getsentry/sentry-javascript/pull/2867

### CODE REVIEW

File: src/platforms/javascript/common/config/index.mdx [hunk 1]
```diff
@@ -8,6 +8,10 @@ Sentry's JavaScript SDK includes many configuration options that are automatical
 - **[Basic Options](./basics/)**
 
   Learn more about how to configure the JavaScript SDK. These options are set when the SDK is first initialized, passed to the `init()` as an object. This page also discusses the Hooks `beforeSend` and `beforeBreadcrumb` as well as transport options.
+  
+- **[Performance](./performance/)**
+
+  Learn more about how to configure Performance for the JavaScript SDK. It provides examples of more advanced use cases.
```
Comment (PeloWriter) [inline src/platforms/javascript/common/config/index.mdx:14]:
```suggestion
  Learn more about how to configure Performance for the JavaScript SDK, including examples of more advanced use cases.
```

File: src/platforms/javascript/common/config/performance.mdx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    }),
+  ],
+  tracesSampleRate: 1.0, // Be sure to lower this in production
+});
+```
+
+### Manual Instrumentation
+
+To manually instrument certain regions of your code, you can create a transaction to capture them.
+This is valid for both JavaScript Browser and Node and works independently of the `BrowserTracing` integration.
```
Comment (PeloWriter) [inline src/platforms/javascript/common/config/performance.mdx:117]:
```suggestion
This is valid for both the JavaScript Browser and Node SDKS, and works independently of the `BrowserTracing` integration.
```

File: src/platforms/javascript/common/config/performance.mdx [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+```javascript
+const transaction = Sentry.startTransaction({ name: "test-transaction" });
+const span = transaction.startChild({ op: "functionX" }); // This function returns a Span
+// functionCallX
+span.finish(); // Remember that only finished spans will be sent with the transaction
+transaction.finish(); // Finishing the transaction will send it to Sentry
+```
+
+Here is a different example. If you want to create a transaction for a user interaction on your page, you need to do the following:
```
Comment (PeloWriter) [inline src/platforms/javascript/common/config/performance.mdx:127]:
```suggestion
This example creates a transaction for a user interaction on your page:
```

**Comment by rhcarvalho** [comment]
@HazAT https://github.com/getsentry/sentry-javascript/pull/2867 is merged, so this should be good to go now.

**Pull request closed without merging**